index.html

```diff
@@ -11,5 +11,25 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="tech-box">
+    <div class="tech-box-content">
```

Comment:
I think that you can remove this extra `div` and work with the styles directly on `tech-box` and its pseudo elements. Give it a try! 😺